NpmVersion tagged package service test failing

:clock11: **When did the problem start?**

Quite a long time ago (at least a month back), but was lost in the noise of the dozens of regularly failing tests we had from David and Librariesio

<!-- Indicate when the problem started -->

:camera: **Live badge**
https://img.shields.io/npm/v/npm/next-8
<!-- Provide a link to the live badge in plain text and markdown.  -->

![](https://img.shields.io/npm/v/npm/next-8)

:wrench: **Is the live badge working?**
Yes
<!-- Indicate whether or not the live badge is working. -->

:link: **CircleCI link**
https://app.circleci.com/pipelines/github/badges/daily-tests/1318/workflows/80397e98-a611-47de-894e-aac3c3a96441/jobs/2220
<!-- Provide a link to the failing test in CircleCI. -->

:lady_beetle: **Stack trace**

```
AssertionError: label mismatch: expected 'npm' to equal 'npm@next'
    at Function._expectField (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:70:51)
    at IcedFrisbyNock.<anonymous> (file:///home/circleci/project/shields/core/service-test-runner/icedfrisby-shields.js:54:26)
    at IcedFrisbyNock.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:954:10)
    at invokeNextHook (node_modules/icedfrisby/lib/icedfrisby.js:1003:24)
    at /home/circleci/project/shields/node_modules/icedfrisby/lib/icedfrisby.js:1017:7
    at new Promise (<anonymous>)
    at IcedFrisbyNock._runHooks (node_modules/icedfrisby/lib/icedfrisby.js:976:12)
    at IcedFrisbyNock.run (node_modules/icedfrisby/lib/icedfrisby.js:1276:20)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Context.<anonymous> (node_modules/icedfrisby/lib/icedfrisby.js:1348:9)
```

:bulb: **Possible solution**
Looks like the npm folks changed their tags from a singular `next` tag to versioned nexts, e.g. `next-7`, so we just need to update our targets 
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

<!-- Love Shields? Please consider donating $10 to sustain our activities:
👉  https://opencollective.com/shields -->

--- a/services/npm/npm-version.service.js
+++ b/services/npm/npm-version.service.js
@@ -34,14 +34,14 @@ export default class NpmVersion extends NpmBase {
     {
       title: 'npm (tag)',
       pattern: ':packageName/:tag',
-      namedParams: { packageName: 'npm', tag: 'next' },
+      namedParams: { packageName: 'npm', tag: 'next-8' },
       staticPreview: this.render({ tag: 'latest', version: '6.3.0' }),
       keywords,
     },
     {
       title: 'npm (custom registry)',
       pattern: ':packageName/:tag',
-      namedParams: { packageName: 'npm', tag: 'next' },
+      namedParams: { packageName: 'npm', tag: 'next-8' },
       queryParams: { registry_uri: 'https://registry.npmjs.com' },
       staticPreview: this.render({ tag: 'latest', version: '7.0.0' }),
       keywords,

--- a/services/npm/npm-version.tester.js
+++ b/services/npm/npm-version.tester.js
@@ -15,8 +15,8 @@ t.create('gets the package version of @cycle/core')
   .expectBadge({ label: 'npm', message: isSemver })
 
 t.create('gets a tagged package version of npm')
-  .get('/npm/next.json')
-  .expectBadge({ label: 'npm@next', message: isSemver })
+  .get('/npm/next-8.json')
+  .expectBadge({ label: 'npm@next-8', message: isSemver })
 
 t.create('gets the correct tagged package version of npm')
   .intercept(nock =>

